tests/customer-resource-embargo-test.js

```diff
@@ -80,6 +80,44 @@ describeApplication('CustomerResourceEmbargo', () => {
         expect(CustomerResourceShowPage.$customEmbargoForm).to.exist;
       });
 
+      describe('then trying to navigate away', () => {
```

Comment:
since this impacts every editable field, and since this will eventually also apply to packages (and even custom resources/packages), i might like to see these tests pulled out into a separate test file instead of dropped into each field-specific test file.  the setup and execution would essentially be the same, but this way it's all in one easy-to-find place.  plus that would allow us to segregate the `sinon` stuff out of the rest of the suite.

Reply:
No more `sinon` stuff. 🙂

I decided to leave in basic tests for each field that ensure editing that specific field and trying to navigate away doesn't go anywhere. But the tests around interacting with the navigation modal have been pulled into the `customer-resource-show` tests. I think we should repeat those tests for packages as well since that might have a different message, or maybe the message will change depending on what's being edited, etc.